src/Grpc/test/InteropTests/Helpers/InteropTestsFixture.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace InteropTests.Helpers
+{
+    public class InteropTestsFixture : IDisposable
+    {
+        private WebServerProcess _process;
+
+        public async Task EnsureStarted(ITestOutputHelper output)
+        {
+            if (_process != null)
+            {
+                return;
+            }
+
+            var webPath = @"C:\Development\Source\AspNetCore\src\Grpc\test\testassets\InteropTestsWebsite\";
```

Comment:
What is the best way to find the client/server projects at runtime?

Reply:
Pass it in via an assembly attribute from MSBuild: https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/test/ProjectTemplates.Tests.csproj#L89-L92